src/terminal.rs

```diff
@@ -29,6 +29,12 @@ where
     B: Backend,
 {
     terminal: &'a mut Terminal<B>,
+
+    /// Where should the cursor be after drawing this frame?
+    ///
+    /// If `None`, the cursor is hidden and its position is controlled by the backend. If `Some((x,
+    /// y))`, the cursor is shown and placed at `(x, y)` after the call to `Terminal::draw()`.
+    resulting_cursor_position: Option<(u16, u16)>,
```

Comment:
```suggestion
    cursor_position: Option<(u16, u16)>,
```
given that there is no other cursor position, I think that we can drop the adjective.

Reply:
I wanted to emphasize that this position will become effective only after the frame is drawn, but I guess stating that in the docstring is enough.